docs/content/topics/operators/deploy.md

```diff
@@ -11,4 +11,348 @@ aliases:
+
+3. Add certificate and key material to the corresponding fields in the chart
+   values file
+    - Set `apiserver.tls.cert` to the certificate value
+    - Set `apiserver.tls.key` to the key value
+    - Set `apiserver.tls.generateSelfSignedCert` to `false`
+
+4. Ensure the [Kubernetes Cert Secret] resource exists, either by creating it
+   yourself or via another method (e.g. [Cert Manager] or another provider)
+    - Set `apiserver.tls.generateSelfSignedCert` to `false`
+
+For a production cluster, we highly recommend options 3 or 4.
+
+[Kubernetes Cert Secret]: charts/brigade/templates/cert-secret.yaml
+[Cert Manager]: https://cert-manager.io/docs/
+
+## Managing inbound traffic via Ingress
+
+By default, the Brigade chart configures the service type for the API server to
+be `LoadBalancer`, which utilizes the cloud provider's infrastructure to
+provision an external IP address. If this is suitable for your needs, you can
+proceed to the next section.  After deployment of Brigade, you'll locate this
+IP for the optional DNS record configuration.
+
+Alternatively, you can configure an [Ingress resource] to be created when
+Brigade is deployed. (We'll assume you've already selected an [Ingress
+Controller] and deployed it to your Kubernetes cluster.)
+
+DNS record creation will be mandatory when using Ingress, so be sure you have a
+domain ready and you are able to create A records under this domain.
+
+All configuration will go under the `apiserver` section of the chart values
+file. Here's the breakdown of the relevant fields, with other nearby sections
+omitted:
+
+```yaml
+apiserver:
+
+  ## This is the DNS hostname you'll set up in a subsequent step
+  ## For example, if you own the example.com domain and you create
+  ## an A record for 'mybrigade', the host will be 'mybrigade.example.com'
+  host: mybrigade.example.com
+
+  ## Ensure tls is enabled for all Brigade's internal components
+  ## For this example, we'll use self-signed certs
+  tls:
+    enabled: true
+    generateSelfSignedCert: true
+
+  ## Here's where we configure ingress
+  ingress:
+    enabled: true
+    ## Annotations specific to the chosen ingress controller go here
+    annotations:
+    tls:
+      enabled: true
+      generateSelfSignedCert: false
+      # cert: base 64 encoded cert goes here
+      # key: base 64 encoded key goes here
+```
+
+Save your updates to `brigade-values2.yaml` and proceed to the next section.
+
+[Ingress resource]: https://kubernetes.io/docs/concepts/services-networking/ingress/
+[Ingress Controller]: https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/
+
+## Third-Party Authentication
+
+The default mode of authentication into Brigade, that of a root user and
+password, is not suitable for a setup involving multiple users.
+
+For production-grade user authentication, Brigade ships with support for
+OIDC-compatible providers (such as [Google Identity Platform] and
```

Comment:
I think the acronym OIDC is only well-known to people with prior OpenID Connect experience. It might be better to shun that acronym in favor of explicitly saying "OpenID Connect."